src/main/java/com/fasterxml/jackson/databind/deser/BasicDeserializerFactory.java

```diff
@@ -1712,7 +1712,10 @@ public JsonDeserializer<?> createEnumDeserializer(DeserializationContext ctxt,
             if (deser == null) {
                 deser = new EnumDeserializer(constructEnumResolver(enumClass,
                         config, beanDesc.findJsonValueAccessor()),
-                        config.isEnabled(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS));
+                        config.isEnabled(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS),
+                        ctxt.isEnabled(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE),
```

Comment:
Hmmmh. This is problematic since `DeserializationFeature`s are meant to be per-call basis; that is, its value may be changed via `ObjectReader` so we cannot assume that value at this point will not change.
This is different from `MapperFeature`s which are per-mapper and cannot be changed.

Put another way, `DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE` needs to be accessed dynamically when needed, not on constructor.

Reply:
Thanks for looking into the PR. 
I have removed the features from constructor in BasicDeserializerFactory.